fastlane/README.md

```diff
@@ -13,66 +13,90 @@ For _fastlane_ installation instructions, see [Installing _fastlane_](https://do
 
 # Available Actions
 
-### check_pods
+### bump
```

Comment:
The order of these actions has changed... Not sure if that's a problem. It does include the actions from `CommonFastfile` though.